### ISSUE TITLE
Cannot enter Game in 4.12.12 in Android

### ISSUE THREAD

**Issue by AutumnPizazz** [yairm210/Unciv#11982]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Game Version

4.12.12

### Describe the bug

**Platform:** Android
**Version:** 4.12.12 (Build 1,025)
**Rulesets:** [Civ V - Gods & Kings, CoeHarMod, Civ V - Vanilla, 5Hex Tileset, Leader Mission 2 Rising Power]
**Last Screen:** `com.unciv.GameStartScreen`

--------------------------------

Device Model: 22021211RC
API Level: 34
System Memory: 11599 MB
    Available (used by Kernel): 2868 MB
    System Low Memory state: false
    Java heap limit: 512 MB
    Java heap free: 444 MB

--------------------------------

**Message:**
```
java.lang.NoSuchMethodError: No static method ofInstant(Lj$/time/Instant;Lj$/time/ZoneId;)Lj$/time/LocalDate; in class Lj$/time/LocalDate; or its super classes (declaration of 'j$.time.LocalDate' appears in /data/app/~~t_8iJ2Tkd3JS3y7Ynk9LzA==/com.unciv.app-2jdH_RscLq-J32gCwyswDg==/base.apk!classes3.dex)
	at com.unciv.logic.HolidayDates$Holidays$Qingming.getByYear(HolidayDates.kt:64)
	at com.unciv.logic.HolidayDates.getHolidayByDate(HolidayDates.kt:159)
	at com.unciv.logic.HolidayDates.getHolidayByDate$default(HolidayDates.kt:155)
	at com.unciv.ui.screens.mainmenuscreen.MainMenuScreen.<init>(MainMenuScreen.kt:123)
	at com.unciv.UncivGame$create$3$1.invokeSuspend(UncivGame.kt:172)
	at com.unciv.UncivGame$create$3$1.invoke(Unknown Source:8)
	at com.unciv.UncivGame$create$3$1.invoke(Unknown Source:4)
	at com.unciv.utils.ConcurrencyKt$launchCrashHandling$1.invokeSuspend(Concurrency.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:190)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:190)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandling$1.invoke(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher.dispatch$lambda$0(Concurrency.kt:190)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher.$r8$lambda$1exBTjTJ644h9iUU94EW5nQQXpM(Unknown Source:0)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$$ExternalSyntheticLambda0.run(Unknown Source:2)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:498)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1577)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1272)

```

### Steps to Reproduce

Enter Game.

### Screenshots

![BUGforAndroid](https://github.com/user-attachments/assets/6cbc6177-2a75-4ff3-a4ea-4e1e06f79eaa)

### Operating System

Android

**Comment by lolafur** [comment]
Sorry to post here, but I ran into a similar problem when I upgraded to version 4.12.12. A complete reinstallation of the game didn't help.

**Comment by yairm210** [comment]
Solved with 4.12.12-patch1
